src/pages/index.jade

```diff
@@ -4,6 +4,7 @@ include ../elements/**/*.jade
 extends ../layouts/default
 
 block meta
+  title Skeleton Example Page
```

Comment:
project-name Page
We need to use "project-name" because that needs to be replaced when creating a new project